src/sentry/api/endpoints/project_group_index.py

```diff
@@ -165,6 +168,14 @@ def validate_assignedTo(self, attrs, source):
             raise serializers.ValidationError('Cannot assign to non-team member')
         return attrs
 
+    def validate(self, attrs):
+        attrs = super(GroupValidator, self).validate(attrs)
+        if 'discard' in attrs and len(attrs) > 1:
```

Comment:
[`len(x)` is O(1) and `x in y` is O(n)](https://wiki.python.org/moin/TimeComplexity) so performance is theoretically better if the order of these expressions is reversed.